dashboard/src/components/Config/AppRepoList/AppRepoForm.tsx

```diff
@@ -614,8 +614,8 @@ export function AppRepoForm(props: IAppRepoFormProps) {
 
       {namespace === kubeappsNamespace && (
         <p>
-          <strong>NOTE:</strong> This App Repository will be created in the "{kubeappsNamespace}"
-          namespace and packages will be available in all namespaces for installation.
+          <strong>NOTE:</strong> This Package Repository will be created in the "{kubeappsNamespace}
+          " namespace and packages will be available in all namespaces for installation.
```

Comment:
Won't HTML assume that new-line there is a word-break? (Not what's wanted in that situation, with the quote, I don't think?)

Reply:
![image](https://user-images.githubusercontent.com/11535726/171066554-5d037624-9f12-4d6d-a53e-ce1db4ab4e69.png)

No, IRL it seems it works as expected. It is a breakline introduced by the linter.